README.md

```diff
@@ -45,3 +48,19 @@ The tests can now be run with nose:
    
 Note that **you cannot run the sls deploy while the virtualenv is active**
 due to how the serverless Python requirements plugin operates.
+
+## Run Test
+
+1. login to AWS console
```

Comment:
Nit: Not sure if you wanted to do uppercase "L" here and uppercase "O" on L62 below, for consistency.